bin/router/src/pipeline/authorization.rs

```diff
@@ -0,0 +1,855 @@
+use crate::pipeline::coerce_variables::CoerceVariablesPayload;
```

Comment:
nit: this file grew a lot and I can see a several areas where it can be splited.